src/sentry/static/sentry/app/views/app/index.tsx

```diff
@@ -198,14 +198,14 @@ class App extends React.Component<Props, State> {
     }
   }
 
-  @keydown('meta+shift+p', 'meta+k')
+  @keydown('meta+shift+p', 'meta+k', 'ctrl+shift+p', 'ctrl+k')
   openCommandPalette(e) {
     openCommandPalette();
     e.preventDefault();
     e.stopPropagation();
   }
 
-  @keydown('meta+shift+l')
+  @keydown('meta+shift+l', 'ctrl+shift+l')
```

Comment:
Actually, let's change `meta` -> `command` as `meta` = "windows" key on Windows

Reply:
I don't think it will make any difference:

https://github.com/glortho/react-keydown/wiki/API-&-Reference#keys
>meta (aka command or cmd)